src/main/resources/db_scripts/createDB.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE quizowanie;
+USE quizowanie;
+
+CREATE TABLE question (
+	quest_id INT NOT NULL,
+	title VARCHAR(128) NOT NULL,
+	description VARCHAR(500),
+	PRIMARY KEY (quest_id)
+);
+
+CREATE TABLE answer (
+	ans_id INT NOT NULL,
+	text VARCHAR(256) NOT NULL,
+	is_correct BOOLEAN NOT NULL,
+	PRIMARY KEY (ans_id)
+);
+
+CREATE TABLE quiz (
+	quiz_id INT NOT NULL,
+	name VARCHAR(128) NOT NULL,
+	maxTime int NOT NULL,
+	PRIMARY KEY (quiz_id)
+);
+
+CREATE TABLE user (
+	user_id INT NOT NULL,
+	username VARCHAR(64) NOT NULL,
+	password BINARY NOT NULL,
```

Comment:
Pole password również może być typu tekstowego, pamiętasz BCrypt z Hackatonu? Tam trzymaliśmy jako tekst i też nie było wiadomo jakie to hasło :)